tools/gcp-broker-provider/bin/gcp-broker.sh

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env bash
+
+# Following script provisions and deprovisions GCP Broker in given namespace.
+#
+# Expected environment variables:
+#   - WORKING_NAMESPACE - name of the namespace where GCP Broker should be installed
+#
+# Input flags:
+#   - action - possible values: provision or deprovision
+
+set -o errexit # exit immediately if a command exits with a non-zero status.
+
+readonly CURRENT_DIR="$(CDPATH= cd -- "$(dirname -- "$0")" && pwd)"
+readonly GCP_SECRET_NAME="gcp-broker-data"
+
+function discoveredUnsetVar() {
+    local discoveredUnsetVar=false
+    for e in WORKING_NAMESPACE; do
+        if [ -z "${!e}" ] ; then
+            echo "ERROR: ${e} is not set"
+            discoveredUnsetVar=true
+        fi
+    done
+
+    if [ "${discoveredUnsetVar}" = true ] ; then
+        exit 1
+    fi
+}
+
+function configureGCloud() {
+    if kubectl get secrets -n ${WORKING_NAMESPACE} | grep -q "${GCP_SECRET_NAME}"; then
+        export GOOGLE_APPLICATION_CREDENTIALS=${CURRENT_DIR}"/sa-key.json"
+
+        projectName=$(kubectl get secret gcp-broker-data -o jsonpath='{ .data.project-name }' | base64 -D)
```

Comment:
base64 in linux does not accept "-D", use "-d" instead

Reply:
FYI: I changed it to full name `--decode` because we want to support both linux and osx